app/src/main/java/org/example/perfEvaluator/PerfTestParams.java

```diff
@@ -0,0 +1,12 @@
+package org.example.perfEvaluator;
+
+import org.example.scheduler.SchedulingPolicy;
+
+/**
+ * The PerfTestParams class encapsulates the parameters for a performance test.
+ */
+public record PerfTestParams(

```

Comment:
A `record` is basically a class with just properties and getters/setters. It's a new shorthand in Java 17 I believe.